Rubberduck.Parsing/Annotations/Concrete/ModuleCleanupAnnotation.cs

```diff
@@ -1,8 +1,23 @@
 ﻿namespace Rubberduck.Parsing.Annotations
 {
     /// <summary>
-    /// Marks a method that the test engine will execute after all unit tests in a test module have executed.
+    /// @ModuleCleanup annotation, marks a procedure that the unit testing engine executes after all tests of a module have completed.
     /// </summary>
+    /// <example>
+    /// <module name="TestModule1" type="Standard Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// '@TestModule
+    /// 
+    /// Private Assert As Rubberduck.AssertClass
+    /// 
+    /// '@ModuleCleanup
+    /// Private Sub ModuleCleanup()
+    ///     Set Assert = Nothing
```

Comment:
Question and a comment: 
Question: Does the presence/effect of  `'@ModuleCleanup` require the preceding presence of `'@TestModule`?
Comment: The other test related annotations include "test" somewhere in the name - except this one.  Would `'@TestModuleCleanup` be more consistent/explicit?

Reply:
`@ModuleCleanup` doesn't look like it's getting flagged as illegal outside of a test module, however it's only ever invoked if it's in a test module, when _that_ test module's tests have all executed.

Renaming this annotation could break VBA unit tests written against Rubberduck 1.0! The annotations were heavily based on MS-Test attributes:

|MS-Test|Rubberduck|
|---|---|
|`[TestClass]`|`@TestModule`|
|`[TestMethod]`|`@TestMethod`|
|`[TestInitialize]`|`@TestInitialize`|
|`[TestCleanup]`|`@TestCleanup`|
|`[ClassInitialize]`|`@ModuleInitialize`|
|`[ClassCleanup]`|`@ModuleCleanup`|

I still think that was a good idea.